jupyter lab --ip nrnb-cn-06 --config /mnt/beegfs/users/zkoch/.jupyter/rps/tmp.kiyiCH.py --no-browser &
[W 2021-10-14 10:57:05.089 ServerApp] Trait, root, not found.
[I 2021-10-14 10:57:10.197 ServerApp] nbclassic | extension was successfully linked.
[I 2021-10-14 10:57:10.780 ServerApp] nbclassic | extension was successfully loaded.
[I 2021-10-14 10:57:10.784 LabApp] JupyterLab extension loaded from /cellar/users/zkoch/miniconda3/lib/python3.9/site-packages/jupyterlab
[I 2021-10-14 10:57:10.785 LabApp] JupyterLab application directory is /mnt/beegfs/users/zkoch/miniconda3/share/jupyter/lab
[I 2021-10-14 10:57:10.791 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-10-14 10:57:10.793 ServerApp] Serving notebooks from local directory: /mnt/beegfs/users/zkoch/cs_clock_proj/scripts/post_training_analyses
[I 2021-10-14 10:57:10.794 ServerApp] Jupyter Server 1.10.2 is running at:
[I 2021-10-14 10:57:10.794 ServerApp] http://nrnb-cn-06:8888/lab?token=...
[I 2021-10-14 10:57:10.794 ServerApp]  or http://127.0.0.1:8888/lab?token=...
[I 2021-10-14 10:57:10.795 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
Unable to connect to VS Code server.
Error in request
[W 2021-10-14 11:02:51.997 LabApp] Could not determine jupyterlab build status without nodejs
[W 2021-10-14 11:02:54.112 ServerApp] 404 GET /api/contents/ChromatinClock.ipynb?content=0&1634234574016 (100.80.213.242): No such file or directory: ChromatinClock.ipynb
[W 2021-10-14 11:02:54.112 ServerApp] No such file or directory: ChromatinClock.ipynb
[W 2021-10-14 11:02:54.113 ServerApp] 404 GET /api/contents/ChromatinClock.ipynb?content=0&1634234574016 (100.80.213.242) 2.50ms referer=http://nrnb-cn-06.ucsd.edu:8888/lab
[I 2021-10-14 11:02:55.426 ServerApp] Kernel started: 842ca420-8493-4cb8-9af8-cafd4f76d943
[I 2021-10-14 11:04:54.825 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:10:54.944 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:12:55.047 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:14:55.131 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:16:55.207 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:28:55.310 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:30:55.646 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:32:55.762 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:34:55.922 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:36:56.083 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:38:56.198 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:40:56.325 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:42:56.425 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:44:56.548 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:46:56.688 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:50:15.143 ServerApp] Kernel interrupted: 842ca420-8493-4cb8-9af8-cafd4f76d943
[I 2021-10-14 11:50:57.260 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:52:57.416 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:54:57.550 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:56:57.652 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 11:58:57.810 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:00:57.970 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:02:58.073 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:06:58.211 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:08:58.389 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:26:58.543 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:28:58.782 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:30:58.982 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:32:59.154 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:38:59.314 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:40:59.440 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:42:59.554 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:44:59.698 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:46:59.839 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:48:59.990 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:51:00.160 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:53:00.300 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:55:00.470 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:57:00.654 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 12:59:00.906 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 13:01:01.112 ServerApp] Saving file at /feature_analysis.ipynb
[W 2021-10-14 13:42:22.681 ServerApp] Replacing stale connection: 842ca420-8493-4cb8-9af8-cafd4f76d943:b51277ee-8539-4059-b10f-c30c9b009118
Task exception was never retrieved
future: <Task finished name='Task-6848' coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /cellar/users/zkoch/miniconda3/lib/python3.9/site-packages/tornado/websocket.py:1100> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/cellar/users/zkoch/miniconda3/lib/python3.9/site-packages/tornado/websocket.py", line 1102, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/cellar/users/zkoch/miniconda3/lib/python3.9/site-packages/tornado/websocket.py", line 1104, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
[W 2021-10-14 13:42:22.744 ServerApp] Got events for closed stream None
[I 2021-10-14 13:42:27.314 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 14:12:29.303 ServerApp] Saving file at /feature_analysis.ipynb
[I 2021-10-14 14:14:29.606 ServerApp] Saving file at /feature_analysis.ipynb
[W 2021-10-14 14:45:22.705 ServerApp] WebSocket ping timeout after 119995 ms.
[W 2021-10-14 14:45:23.507 ServerApp] WebSocket ping timeout after 119900 ms.
[I 2021-10-14 14:45:28.509 ServerApp] Starting buffering for 842ca420-8493-4cb8-9af8-cafd4f76d943:544801e3-86da-41a3-883a-f67cd6d28eb9
[W 2021-10-14 15:37:00.166 ServerApp] WebSocket ping timeout after 119993 ms.
[I 2021-10-14 15:37:05.185 ServerApp] Starting buffering for 842ca420-8493-4cb8-9af8-cafd4f76d943:b51277ee-8539-4059-b10f-c30c9b009118
slurmstepd: error: *** JOB 429098 ON nrnb-cn-06 CANCELLED AT 2021-10-15T10:29:11 ***
[C 2021-10-15 10:29:11.089 ServerApp] received signal 15, stopping
[I 2021-10-15 10:29:11.096 ServerApp] Shutting down 2 extensions
[I 2021-10-15 10:29:11.097 ServerApp] Shutting down 1 kernel
[I 2021-10-15 10:29:11.098 ServerApp] Kernel shutdown: 842ca420-8493-4cb8-9af8-cafd4f76d943
[I 2021-10-15 10:29:11.300 ServerApp] Shutting down 0 terminals
